database/store.py

```diff
@@ -0,0 +1,174 @@
+"""
+Data Access Object
+A data access object (DAO) is a pattern that provides an abstract interface
+to some type of database or other persistence mechanism.
+"""
+import datetime
+import uuid
+
+from database.initial_data import initial_application
+from database.initial_data import initial_application_store_state
+from dateutil import parser as date_parser
+from dateutil.tz import UTC
+from slugify import slugify
+from operator import itemgetter
+from distutils.util import strtobool
+from external_data.data import get_round
+
+
+class ApplicationDataAccessObject(object):
+    """
+    return all funds (replace the date object with a string to send)
```

Comment:
this is just me I'm sure but i dont get what this is saying?

Reply:
I think this is an old dev comment left in and moved here in error - removed